api/v1/testkube.yaml

```diff
@@ -117,15 +117,15 @@ paths:
           content:
             application/problem+json:
               schema:
-                type: object
+                type: array
                 items:
                   $ref: "#/components/schemas/Problem"
         502:
           description: problem communicating with kubernetes cluster
           content:
             application/problem+json:
```

Comment:
I think we don't return `application/problem+json` in case of an error but regular `application/json`.
Should we change the content type for errors here, or should we change the content type in controllers to return `application/problem+json`?